openfecli/fetchables.py

```diff
@@ -5,6 +5,20 @@
 _EXAMPLE_NB_BASE = ("https://raw.githubusercontent.com/"
                     "OpenFreeEnergy/ExampleNotebooks/main/")
 
+RBFE_SHOWCASE = URLFetcher(
+    resources=[
+        (_EXAMPLE_NB_BASE + "openmm_rbfe/inputs/", "ligands.sdf"),
+        (_EXAMPLE_NB_BASE + "openmm_rbfe/inputs/", "Jnk1_ligands.sdf"),
+        (_EXAMPLE_NB_BASE + "openmm_rbfe/inputs/", "181L_mod_capped_protonated.pdb"),
+        (_EXAMPLE_NB_BASE + "openmm_rbfe/inputs/", "benzene.sdf"),
+        (_EXAMPLE_NB_BASE + "openmm_rbfe/inputs/", "styrene.sdf"),
+    ],
+    short_name="rbfe-showcase",
+    short_help="Inputs needed for the RBFE Showcase Notebook",
+    section="Tutorials",
```

Comment:
Seems to me that, without the actual notebook, this is only intended for internal use. If so, change the section name (to anything, really, but I default to using `"hidden"`) so that it doesn't show in help.